classes/class-wc-connect-taxjar-integration.php

```diff
@@ -657,16 +754,31 @@ public function calculate_tax( $options = array() ) {
 			);
 
 			// Add line item tax rates
-			foreach ( $this->line_items as $line_item_key => $line_item ) {
+			foreach ( $taxes['line_items'] as $line_item_key => $line_item ) {
 				$line_item_key_split = explode( '-', $line_item_key );
```

Comment:
The TaxJar plugin has since renamed this to `$line_item_key_chunks`, let's port that over too. See: https://github.com/taxjar/taxjar-woocommerce-plugin/blob/master/includes/class-wc-taxjar-integration.php#L350